### PR TITLE
Added Dreadrise search

### PR DESCRIPTION
**Pull request by Wizzerinus**
idk if this is needed but why not

if youre pulling this set `dreadrise_url` configuration key to `http://localhost:3001` on production to reduce the network load

### CODE REVIEW

File: discordbot/command.py [hunk 1]
```diff
@@ -257,3 +257,6 @@ async def single_card_text(self, c: Card, f: Callable, show_legality: bool = Tru
     async def post_cards(self, cards: List[Card], replying_to: Optional[Member] = None, additional_text: str = '') -> None:
         # this feels awkward, but shrug
         await post_cards(self.bot, cards, self.channel, replying_to, additional_text)
+
+    async def post_no_matches(self) -> None:
+        await post_no_cards(self.channel)
```
Comment (bakert) [inline discordbot/command.py]: What does this do? Sounds like `post_no_cards` should be renamed to `post_nothing`?

File: discordbot/commands/drc.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
+from discord.ext import commands
```
Comment (bakert) [inline discordbot/commands/drc.py:1]: This is cool.

File: discordbot/command.py (hunk 1 above)
Comment (Wizzerinus) [reply discordbot/command.py]: post_no_cards was a function before but sure i can do that

File: discordbot/commands/drc.py (hunk 2 above)
Comment (Wizzerinus) [reply discordbot/commands/drc.py:1]: Copied from the scryfall script

Comment (bakert) [reply discordbot/commands/drc.py:1]: Sorry that comment was meant for the whole file. I am just appreciating your addition :)

File: discordbot/command.py (hunk 1 above)
Comment (bakert) [reply discordbot/command.py]: What I am suggesting is that `post_no_cards` be renamed to `post_nothing` and that each callsite for `post_no_cards` and `post_no_matches` be replaced by a call to `post_nothing`.

Comment (Wizzerinus) [reply discordbot/command.py]: done

File: discordbot/commands/drc.py (hunk 2 above)
Comment (Wizzerinus) [reply discordbot/commands/drc.py:1]: ah haha got you

**Pull request merged**